lib/renderer.js

```diff
@@ -63,7 +63,12 @@ const secondsToTime = (s) => {
 let currentTime = 0
 
 $('#increase-time').on('click', () => {
-  currentTime = currentTime + 10
+  currentTime = currentTime + 60
+  timerDiv.innerHTML = secondsToTime(currentTime)
+})
+
+$('#decrease-time').on('click', () => {
```

Comment:
![](https://media.giphy.com/media/l0MYOUI5XfRk4LLWM/giphy.gif)

Great job! We have time!